Примите фичи на осмотр

breligor opened the pull request.

breligor committed:
Примите фичи на осмотр

6ygga commented on index.html:
```diff
@@ -0,0 +1,62 @@
+        <button id="firstButton">Нажми меня</button>
+    </div>
+    <div class="task-container" id="secondDiv">
+        <h1>Task2</h1>
```
h1 может быть только 1 на странице
остальные минимум h2

6ygga commented on index.html:
```diff
@@ -0,0 +1,62 @@
+        <button  id="thirdButton">Нажми меня</button>
+        <p>Исходный массив ['Капуста', 'Репа', 'Редиска', 'Морковка']</p>
+    </div>
+    <div class="task-container" id="fourthDiv">
```
id="my-Id"
убирай кэмелкейз в айдишках

6ygga commented on index.html:
```diff
@@ -0,0 +1,62 @@
+    </div>
+    <div class="task-container" id="fourthDiv">
+        <h1>Task4</h1>
+        <button  id="fourthButton">JOIN</button>
```
перед id по 2 пробела, тоже ошибка

6ygga commented on scrypt_1.js:
```diff
@@ -0,0 +1,17 @@
+const obj={
```
пробелы перед и после  =

6ygga commented on scrypt_1.js:
```diff
@@ -0,0 +1,17 @@
+    'Москва': 'Россия',
+    'Киев': 'Украина',
+}
+document.querySelector("#firstButton").onclick = ()=>
```
пробел после =>

6ygga commented on scrypt_1.js:
```diff
@@ -0,0 +1,17 @@
+        let message= key +' - это '+ obj[key]+'!';
+        //вставка на страницу
+        let paragraph = document.createElement("p");
+        let firstDiv= document.getElementById('firstDiv');
```
опять пробел... 
просмотри все файлы сам на аккуратность с пробелами

6ygga replied:
ты постоянно получаешь этот элемент, а нужно его получить 2 раз и потом просто использовать полученную ссылку

6ygga commented on scrypt_2.js:
```diff
@@ -0,0 +1,16 @@
+        for (let i=0; i<7; i++)
+        {   //вставка на страницу
+            let paragraph = document.createElement("p");
+            let secondDiv= document.getElementById('secondDiv');
```
здесь тоже вынеси получение этого элемента из цыкла

6ygga commented on scrypt_3.js:
```diff
@@ -0,0 +1,34 @@
+    function copyArr(){
+        clonedArr = vegetables.slice();
+        //проверка типа
+        console.log(Array.isArray(clonedArr));
```
это не проверка и лишний консольлог
убирай или сделай проверку

6ygga commented on scrypt_4.js:
```diff
@@ -0,0 +1,41 @@
+        {
+        const str=arr.join(',')
+        //проверка типа
+        console.log(typeof str);
```
где проверка?

6ygga commented on scrypt_5.js:
```diff
@@ -0,0 +1,23 @@
+        //перебор массива с проверкой на нечетность
+        for (let i=0; i<arr.length; i++)
+        {
+            if((arr[i]%2==true) && (arr[i+1]%2==true))
```
пробелы....
(arr[i] % 2 == true)

6ygga commented on scrypt_5.js:
```diff
@@ -0,0 +1,23 @@
+        for (let i=0; i<arr.length; i++)
+        {
+            if((arr[i]%2==true) && (arr[i+1]%2==true))
+            {res+=arr[i]+':'}
```
{ res += arr[i] + ':' }

6ygga commented on scrypt_5.js:
```diff
@@ -0,0 +1,23 @@
+         }
+        //вставка на страницу
+        let paragraph= document.createElement("p");
+        let fifthDiv= document.getElementById('fifthDiv');
```
выноси из цикла

6ygga commented on scrypt_6.js:
```diff
@@ -0,0 +1,11 @@
+const today = new Date().getDay() - 1; //сегодняшний день
+for (let i = 0; i < 7; i++) {
+      i === today ?
+       day.innerHTML += `<i>${thisWeek[i]}</i><br>`:
```
используй стили для выделения
почитай как добавлять и убирать класс

6ygga commented on scrypt_7.js:
```diff
@@ -0,0 +1,7 @@
+function removeDuplicates() {
+    const uniqArr = Array.from(new Set(arrDuplicates.map(_ => _.toLowerCase())));
+        const paragraph = document.querySelector('#paragraph');
+        paragraph.innerHTML = 'Уникальный массив '+'['+ uniqArr+']';
```
используй template string

6ygga commented on scrypt_8.js:
```diff
@@ -0,0 +1,8 @@
+function union () {
+    let uniqArr = Array.from(new Set([...array1, ...array2]));
+    const paragraph = document.querySelector('#text');
+    paragraph.innerHTML = 'Уникальный массив ' + '[' + uniqArr + ']';
```
replace with template string

breligor committed:
update index.html

убрал кэмелкейс в айди, убрал лишние h1, убрал пробелы

breligor committed:
changes

пробелы, id

breligor commented:
closed

breligor committed:
changes2

использовал стили для выделения с помощью добавления класса, использовал templare string
